docs/docs/create-transformer-plugin.md

```diff
@@ -2,7 +2,54 @@
 title: Create a Transformer Plugin
 ---
 
-This is a stub. Help our community expand it.
+Transformer plugins are responsible for "transforming" data provided by
+[source plugins](/docs/create-source-plugin) into new nodes and/or node fields.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Transformer plugin example
+
+The [`gatsby-transform-remark`](/packages/gatsby-transform-remark/) plugin
+"transforms" markdown nodes (fetched for example from the filesystem source plugin) into html and provides additional node details such as for example the `timeToRead` the `tableOfContents` and the `excerpt`.
+
+## What do transformer plugins do?
+
+Each transformer plugin is responsible for "enhancing" the existing source nodes they
+are provided to provide additional information. Therefore, you'll often use both source plugins and transformer plugins in your Gatsby sites. Transformer plugins can be heavily opinionated since they choose they way they transform their source.
```

Comment:
I think there should be a new paragraph starting with: 

> Transformer plugins can be heavily opinionated

And the paragraph could include answers to these questions: Do you have an example of how transformer plugins can be heavily opinionated? Why is it important for the reader to know that information?

Reply:
Well basically was reading the source plugins documentation to get a bit of inspirations and this is why i included the phrase. maybe its best to avoid this phrase entirely, in the end it does not seem like important to the reader.

Reply:
Oh sure, we could take it out.